src/chrome/content/rules/derstandard.at.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+	Mismatch:
+		- meinparlament
+		- parship
+		- text
+		- tv
+
+	Problematic:
+		- redirect on some pages, only css, images and js are working
+-->
+<ruleset name="derStandard.at (partial)">
+	<target host="derstandard.at"/>
+	<target host="www.derstandard.at"/>
+	<target host="epaper.derstandard.at"/>
+	<target host="images.derstandard.at"/>
+	<target host="mobil.derstandard.at"/>
+
+	<test url="http://derstandard.at/ajax/bottomnavinfo.ashx" />
+	<test url="http://mobil.derstandard.at/ajax/bottomnavinfo.ashx" />
+	<test url="http://www.derstandard.at/ajax/bottomnavinfo.ashx" />
+	<test url="http://derstandard.at/css/mobile/base.css" />
+	<test url="http://mobil.derstandard.at/css/mobile/base.css" />
+	<test url="http://www.derstandard.at/css/mobile/base.css" />
+	<test url="http://derstandard.at/img/ui/searchLupe.gif" />
+	<test url="http://mobil.derstandard.at/img/ui/searchLupe.gif" />
+	<test url="http://www.derstandard.at/img/ui/searchLupe.gif" />
+	<test url="http://derstandard.at/js/fixedfixed.js" />
+	<test url="http://mobil.derstandard.at/js/fixedfixed.js" />
+	<test url="http://www.derstandard.at/js/fixedfixed.js" />
+	<test url="http://derstandard.at/wetter/Content/img/search-icon2.png" />
+	<test url="http://mobil.derstandard.at/wetter/Content/img/search-icon2.png" />
+	<test url="http://www.derstandard.at/wetter/Content/img/search-icon2.png" />
+
+	<rule from="^http://(www\.)?derstandard\.at/(ajax|css|img|js|wetter/Content/img)/"
+		to="https://derstandard.at/$1/"/>
+	<rule from="^http://mobil\.derstandard\.at/(ajax|css|img|js)/"
+		to="https://mobil.derstandard.at/$1/"/>
+	<rule from="^http://epaper.derstandard.at/"
```

Comment:
Thanks for the feedback. The test server was down today, so I wasn't able to continue.